app/Meta.py

```diff
@@ -0,0 +1,564 @@
+"""
+This is the Meta module and supports all the REST actions for the yaba.yaml
+"""
+
+# Importing modules
+import os
+
+from flask import make_response,abort,Response,jsonify
+from flask import json
+import pandas as pd
+import json
+from db import *
+import geopandas as gpd
+from werkzeug import secure_filename,FileStorage
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.exc import OperationalError
+import traceback
+import logging
+from shapely import geos, wkb, wkt
+from shapely.geometry.multipolygon import MultiPolygon
+
+
+
+UPLOAD_FOLDER = 'temp'
+
+
+
+def save_tempFile(File):
+    # Validate that what we have been supplied with is infact a FileStorage
+    if not isinstance(File, FileStorage):
+        raise TypeError("Storage must be a werkzeug.FileStorage")
+
+    # Sanitise the filename
+    a_file_name = secure_filename(File.filename)
+
+    # Build target
+    a_file_target = os.path.join(os.getcwd(),UPLOAD_FOLDER,a_file_name)
```

Comment:
Suggest moving the construct 'os.getcwd(),UPLOAD_FOLDER' to where UPLOAD_FOLDER is defined; perhaps something like UPLOAD_PATH=os.path.join(os.getcwd(),UPLOAD_FOLDER)